tools/topology/sof-icl-rt711-rt1308-rt715.m4

```diff
@@ -0,0 +1,145 @@
+#
+# Topology for Icelake with rt711 + rt1308 (x2) + rt715.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include ICL DSP configuration
+include(`platform/intel/icl.m4')
+include(`platform/intel/dmic.m4')
```

Comment:
We may not need dmic.m4 since there is no dmic in this file now.

Reply:
good catch, will fix. thanks!

Reply:
updated now